queen-of-versions/src/main/java/co/infinum/queenofversions/InAppUpdateProcessCallback.java

```diff
@@ -21,4 +21,6 @@
     void onInstalling();
 
     void onRequiresUI();
+
+    void onWrongVersion();
```

Comment:
I believe "wrong version" is unhelpful name. Shouldn't `updateUnavailable` be better?

Reply:
Yeah, I agree with you. I named that in a hurry :)